test/DetOne.lean

```diff
@@ -0,0 +1,20 @@
+import Mathlib.LinearAlgebra.Determinant
+
+/-- see https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/kernel.20deep.20recursion.20detected -/
```

Comment:
Can you add a brief comment to this file, explaining what this test is testing? It's fine to refer to zulip for the details, but it would be great if I can get an idea *without* having to read zulip first. Thanks!

Reply:
Added

Reply:
Thanks!